test/viewmodels/helpers/CheckAnswersConsigneeHelperSpec.scala

```diff
@@ -176,6 +183,17 @@ class CheckAnswersConsigneeHelperSpec extends SpecBase with MovementSubmissionFa
           checkAnswersConsigneeHelper.summaryList() mustBe SummaryList(rows = expectedSummaryListRows)
         }
 
+      "the Consignee ERN needs to be updated" in new Setup(testGbWarehouseErn, ernNeedsUpdatingUserAnswers) {
+
+        val expectedSummaryListRows: Seq[SummaryListRow] = Seq(
+          ConsigneeBusinessNameSummary.row(true)(fakeDataRequest, msgs),
+          consigneeExciseSummary.row(true)(fakeDataRequest, msgs),
+          ConsigneeAddressSummary.row(true)(fakeDataRequest, msgs)
+        ).flatten
+
+        checkAnswersConsigneeHelper.summaryList() mustBe SummaryList(rows = expectedSummaryListRows)
+      }
```

Comment:
I left a comment about this on Ladon's PR, I'm not sure on the value of this test. It doesn't distinguish visibly from other tests (and the summary list logic has been tested in its own spec)

Reply:
Can't remember changing this one, I think I've got some stuff leftover from Ladon's branch - will remove